src/WorkflowManager/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -659,8 +681,33 @@ private async Task<string[]> GetArtifactValues(WorkflowRevision workflow, Workfl
 
         private async Task HandleHl7ExportAsync(WorkflowRevision workflow, WorkflowInstance workflowInstance, TaskExecution task, string correlationId)
         {
-            // create message. send
+            var (exportList, artifactValues) = await GetExportsAndArtifcts(workflow, workflowInstance, task, correlationId);
+
+            if (exportList is null || artifactValues is null)
+            {
+                return;
```

Comment:
Doesn't seem like there's any logging on this route? might it be worth adding some kind of log here? imagine it could be annoying if you put a HL7 export task in and it just returns no result and wouldn't know why?